FEATURES.md

```diff
@@ -0,0 +1,175 @@
+* Start-time as stored by `markEvaluationStartTime` is used; end-time = now()
+* README contains an example of how to call this function within a Jenkinsfile (e.g., after tests)
+* Evaluation is triggered within Keptn (verifiable in Keptn Bridge)
+
+
+### User Story 7: Labels to the evaluation event
+
+**Goal**: A user should be able to add labels to the evaluation event that is sent for triggering an evaluation. Also, certain default labels should be added, e.g.:
+
+```
+        |    "labels": {
+        |      "jobname" : "${JOB_NAME}",
+        |      "buildNumber": "${BUILD_NUMBER}",
+        |      "joburl" : "${BUILD_URL}"
+        |    },
+```
+
+**DoD**:
+* Shared library code for triggering an evaluation contains default labels
+* Shared library code for triggering an evaluation takes labels as parameter, and merges it with default labels
+* README contains an example of how to call this function within a Jenkinsfile
+* Labels appear in Keptn Bridge (verifiable in Keptn Bridge)
+
+
+### User Story 8: Wait for evaluation.finished
+
+**Goal**: A user should be able to wait for Keptn to generate the evaluation.finished event, and put the result as a Jenkins pipeline result.
+
+**Example**:
+
+```
+    stage('Wait for Result') {
+        echo "Waiting until Keptn is done and returns the results"
+        def result = keptn.waitForEvaluationFinishedEvent setBuildResult:true, waitTime:5
+        echo "${result}"
+    }
+```
+
+
+**DoD**:
+* Shared library code contains a function to wait for an evaluation.finished event (based on the shkeptncontext of evaluation.triggered)
+* Function can be configured with a waitTime
+* Function can be configured to set a build result for the Jenkins Pipeline
+* README contains an example of how to call this function within a Jenkinsfile
+
+
+### User Story 9: Trigger a delivery
+
+**Goal**: A user should be able to trigger a delivery within their Jenkinsfile.
+
+**Example**:
+```
+    stage('Docker build/push') {
+      docker.withRegistry('https://index.docker.io/v1/', 'dockerhub') {
+        def app = docker.build("${image_name}:${commit_id}", '.').push()
+      }
+    }
+
+    stage('Trigger Delivery') {
+        def keptnContext = keptn.sendDeliveryTriggeredEvent image:"${image_name}:${commit_id}"
+    }
+```
+
+**DoD**:
+* Shared library code contains a function to trigger a delivery in Keptn
+* Shared library code for triggering a delivery takes a container/image as a parameter
+* README contains an example of how to call this function within a Jenkinsfile
+* Delivery is triggered within Keptn (verifiable in Keptn Bridge)
+
+
+### User Story 10: Call Jenkins pipeline from Keptn
+
+**Goal**: A user should be able to call a Jenkins pipeline from Keptn.
+
+**Notes**: This is handled via [Keptn webhook-service](https://keptn.sh/docs/0.10.x/integrations/webhooks/).
+
+**DoD**:
+* Example/documentation provided (e.g., in Keptn docs or on Keptn artifacthub) how to call Jenkins from Keptn.
```

Comment:
Let`s focus on building integrations for ArtifactHub and not Keptn docs.

 (e.g., ~~in Keptn docs or~~ on Keptn ArtifactHub) 

DoD: 
- https://keptn.sh/docs/0.10.x/integrations/webhooks/ transferred into ArtifactHub entry (like: https://artifacthub.io/packages/keptn/keptn-integrations/slack)

Reply:
Same as with the entries before, currently the documentation is provided in Keptn Docs.